packages/core/src/cloudProvider/CloudProviderRegistry.ts

```diff
@@ -13,37 +13,11 @@ export interface ICloudProviderConfig {
   [attribute: string]: any;
 }
 
-class Providers {
-  private providers: Array<{ cloudProvider: string; config: ICloudProviderConfig }> = [];
-
-  public set(cloudProvider: string, config: ICloudProviderConfig): void {
-    // The original implementation used a Map, so calling #set could overwrite a config.
-    // The tests depend on this behavior, but maybe something else does as well.
```

Comment:
There was some logic that was added for kubernetes provider v1 and v2 that made some of this Providers logic useful, but those k8s providers have since been collapsed into a single version.